_includes/app/gorm-basic-sample.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	// Import GORM-related packages.
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+)
+
+// Account is our model, which corresponds to a database table.
+type Account struct {
+	ID      int `gorm:"primary_key"`
+	Balance int
+}
+
+func main() {
+	// Connect to CockroachDB.
```

Comment:
What do you think about the following?

```
//Connect to the "bank" database as the "maxroach" user.
```

It's closer to, but even more explicit than, the comments in the driver examples...